src/js/components/message.jsx

```diff
@@ -73,15 +70,9 @@ export class MessageComponent extends Component {
             <img className={ avatarClass.join(' ') }
                  src={ avatarUrl } />;
 
-        const extraSpacingStyle = {
-            marginBottom: '5px',
-            display: 'inline-block'
-        };
-
         const textPart = hasText && <TextMessage {...this.props}
```

Comment:
why is text not getting `sk-message-item sk-message-text-last-item-long-classname` thing?